src/components/ThemeSelect/components/ThemeDropDown/ThemeDropDown.jsx

```diff
@@ -0,0 +1,23 @@
+import ThemeItem from '../ThemeItem/ThemeItem';
+import classes from './ThemeDropDown.module.scss';
+import { getThemesList } from '../../../../config/themeConfig';
+import ThemeContext from '../../../../providers/themeContext';
+
+function ThemeDropDown({ currentTheme, selectTheme }) {
+  const themeList = getThemesList();
```

Comment:
Again: does it need to be in the scope of this component? HINT: no

Reply:
I moved it outside of component scope.